<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileHunterMWClass</class>
 <widget class="QMainWindow" name="FileHunterMWClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileHunter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="m_pSrcLineEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="m_pSrcButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="m_pMaskEdit">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>*.*</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="m_pDstLineEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="m_pDstButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>57</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>0</y>
         <width>241</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Показывать</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="m_pEqualButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="FileHunterMW.qrc">
            <normaloff>:/FileHunterMW/Resources/equal.png</normaloff>:/FileHunterMW/Resources/equal.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pNoEqualButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="FileHunterMW.qrc">
            <normaloff>:/FileHunterMW/Resources/no_equal.png</normaloff>:/FileHunterMW/Resources/no_equal.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pRightLeftButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="FileHunterMW.qrc">
            <normaloff>:/FileHunterMW/Resources/right_left.png</normaloff>:/FileHunterMW/Resources/right_left.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pLeftRightButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="FileHunterMW.qrc">
            <normaloff>:/FileHunterMW/Resources/left_right.png</normaloff>:/FileHunterMW/Resources/left_right.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pNotFoundButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>не найденные</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>151</width>
         <height>50</height>
        </rect>
       </property>
       <property name="title">
        <string>Поиск</string>
       </property>
       <widget class="QPushButton" name="m_pSearchButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>18</y>
          <width>70</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="m_pSrcDirRadioBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>70</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Папка</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="m_pSrcFileRadioBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>14</y>
          <width>70</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Файл</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>408</x>
         <y>0</y>
         <width>81</width>
         <height>50</height>
        </rect>
       </property>
       <property name="title">
        <string>Копировать</string>
       </property>
       <widget class="QPushButton" name="m_pLeftRightCopyButton">
        <property name="geometry">
         <rect>
          <x>45</x>
          <y>16</y>
          <width>26</width>
          <height>26</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="FileHunterMW.qrc">
          <normaloff>:/FileHunterMW/Resources/left_right.png</normaloff>:/FileHunterMW/Resources/left_right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="m_pRightLeftCopyButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>16</y>
          <width>26</width>
          <height>26</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="FileHunterMW.qrc">
          <normaloff>:/FileHunterMW/Resources/right_left.png</normaloff>:/FileHunterMW/Resources/right_left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="m_pTable">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>21</number>
      </attribute>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>&lt;=&gt;</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_pOpenFileAction">
   <property name="text">
    <string>Открыть файл</string>
   </property>
   <property name="toolTip">
    <string>SVN: Открыть файл</string>
   </property>
  </action>
  <action name="m_pSvnUpdateAction">
   <property name="text">
    <string>SVN: Обновить</string>
   </property>
   <property name="toolTip">
    <string>SVN: Обновить</string>
   </property>
  </action>
  <action name="m_pSvnFixAction">
   <property name="text">
    <string>SVN: Фиксировать</string>
   </property>
  </action>
  <action name="m_pSvnLogAction">
   <property name="text">
    <string>SVN: Журнал</string>
   </property>
  </action>
  <action name="m_pSvnRepobrowserAction">
   <property name="text">
    <string>SVN: Обозреватель хранилища</string>
   </property>
  </action>
  <action name="m_pMergeAction">
   <property name="text">
    <string>Сравнить файлы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileHunterMW.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pSrcButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotSrcBrowseButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pDstButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotDstBrowseButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pTable</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotItemClicked(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pSearchButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pEqualButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotEqualClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>-19</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pNoEqualButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotEqualClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>-35</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRightLeftButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotEqualClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>-61</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pLeftRightButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotEqualClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>-57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pNotFoundButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotEqualClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>-27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRightLeftCopyButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>-26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pLeftRightCopyButton</sender>
   <signal>clicked()</signal>
   <receiver>FileHunterMWClass</receiver>
   <slot>slotCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>-49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSrcBrowseButton()</slot>
  <slot>slotDstBrowseButton()</slot>
  <slot>slotSearch()</slot>
  <slot>slotItemClicked(QTableWidgetItem*)</slot>
  <slot>slotEqualClicked()</slot>
  <slot>slotCopy()</slot>
 </slots>
</ui>
